### DISCUSSION TITLE
Any way to automatically output a method's return value?

### DISCUSSION THREAD

**Discussion by clirette** [waf/CSharpRepl#244] [Q&A] [answered]
I'd like to see the output from a method automatically without having to resort to my current workaround - assigning to a variable then outputting _that_ value.

Here's what I mean:

![image](https://user-images.githubusercontent.com/25960323/232100719-169399d7-2bf6-40fc-a498-7c66747470a2.png)

Is there a faster way to see the output of a method?

**Comment by kindermannhubert** [comment, marked as answer]
Just skip the `;` at the end.
Expression ending with `;` is a statement and it does not have any return value.

**Reply by clirette** [reply]
🤦‍♂️ Thank you!
